data/jibauni/jibauni missions.txt

```diff
@@ -0,0 +1,575 @@
+			choice
+				`	"Goodbye."`
+					goto end
+				`	"This is... a lot of money for a single hop."`
+				`	"You do realize that one of the people who helped you load your cargo on <origin> already paid me something?"`
+					goto dockpayment
+			`	"I do not know what is normally the amount for space travel. I have not been out of <origin>'s atmosphere for many years," replies the Jibauni.`
+			choice
+				`	"The normal payment is much less. Take five thousand credits back."`
+				`	"That makes sense. Goodbye!"`
+					goto end
+			action
+				payment -5000
+				set "jibauni: paid an-lo and re-sop back"
+			`	"Shukran jezeelem," says An-lo. The translation device makes no attempt to translate these words.`
+				goto end
+			label dockpayment
+			`	"That was kind of him," says Re-sop. "But why do you mention it?"`
+			choice
+				`	"Because I wanted to pay you that money back."`
+				`	"No reason, just curious."`
+					goto end
+			action
+				payment -1000
+				set "jibauni: paid an-lo and re-sop back"
+			`	"Very kind of you," she says.`
+			label end
+			`	Both Jibauni exit the <ship> with their bags, and you go about your business.`
+
+
+mission "Jibauni: Ru-win 0"
+	minor
+	name "Ru-win to <planet>"
+	description "Take Ru-win, who you talked to on <origin>, to <planet>."
+	source "Kanlil"
+	destination "Baitoon"
+	passengers 1
+	to offer
+		has "language: Jibauni"
+		has "Jibauni Culture: Hajay Intersystem: declined"
+		not "jibauni: didn't ask about hajay intersystem"
+		random < 16
+	on offer
+		conversation
+			`As you walk into the spaceport, one Jibauni catches your eye. They seem to be looking for someone, and are glancing all over the place.`
+			choice
+				`	(Go and see what is going on.)`
+				`	(Stay here.)`
+					goto stay
+				`	(Leave.)`
+					decline
+			`	You walk toward the Jibauni, and ask, "What is happening?" As you notice that it's Ru-win, she gives a start and replies, "At last! The human captain, <first>! There you are! Come with me over here to a more quiet place, I need to ask you something." She pushes you over to a bench.`
+				goto question
+			label stay
+			`	After a bit longer of looking around, the Jibauni heads your way, and you notice she is Ru-win. "Captain <last>! I need to ask you something! Come over to this seating area with me?" She pushes you over to a bench.`
+			label question
+			`	As you sit down, she says, "The thing I am looking for help with is transport to Baitoon. I need to get there and buy a ship, fast, before I run out of... well, let me tell you the story from the beginning. A while after you saw me, on Kanlil, my ship ran into snags. When I took it to the repair shop, they said they could not fix it. I sold it to them for a mere trifle."`
+			`	"So, what do you say? Will you take me to Baitoon? Do realize that I do not currently have the money to pay you though."`
+			choice
+				`	"Sure, I'll take you."`
+				`	"No, sorry. I don't go around giving rides to random strangers."`
```

Comment:
The only no option is one where you are kind of insulting. Maybe have another no that is less insulting? You could even do a defer with, "Sorry, I am busy right now."

Reply:
Well, maybe something like. "Not interested."?